OpenMcdf.Ole/OlePropertiesContainer.cs

```diff
@@ -19,14 +19,26 @@ public class OlePropertiesContainer
 
     public OlePropertiesContainer? UserDefinedProperties { get; private set; }
 
+    /// <summary>
+    /// Gets the type of the container.
+    /// </summary>
     public ContainerType ContainerType { get; }
-    private Guid? FmtID0 { get; }
+
+    /// <summary>
+    /// Gets the FMTID of the properties container.
+    /// </summary>
+    public Guid FMTID0 { get; }
 
     public PropertyContext Context { get; }
 
     private readonly List<OleProperty> properties = new();
     internal Stream? cfStream;
```

Comment:
A thought when looking at this - is there actually a need to keep the stream as a member?